src/components/Personalization/helper/dom/getElementById.js

```diff
@@ -13,9 +13,9 @@ governing permissions and limitations under the License.
 /**
  * Returns an array of matched DOM nodes.
  * @param {String} id
- * @param {Node} doc, defaults to document
+ * @param {Node} context, defaults to document
```

Comment:
It's really not a big deal, but if you're wanting to stick to JSDoc, the way you would show an optional param with a default value would be like this: `@param {Node} [context=document]`

Reply:
Will change, thanks!